models/error.go

```diff
@@ -1627,6 +1627,22 @@ func (err ErrPullRequestHasMerged) Error() string {
 		err.ID, err.IssueID, err.HeadRepoID, err.BaseRepoID, err.HeadBranch, err.BaseBranch)
 }
 
+// Error is the error implementation
+func (err ErrPullRequestAlreadyScheduledToAutoMerge) Error() string {
+	return fmt.Sprintf("pull request is already scheduled to auto merge when checks succeed [pull_id: %d]", err.PullID)
+}
+
+// ErrPullRequestAlreadyScheduledToAutoMerge represents a "PullRequestHasMerged"-error
+type ErrPullRequestAlreadyScheduledToAutoMerge struct {
+	PullID int64
+}
```

Comment:
```suggestion
// ErrPullRequestAlreadyScheduledToAutoMerge represents a "PullRequestHasMerged"-error
type ErrPullRequestAlreadyScheduledToAutoMerge struct {
	PullID int64
}

func (err ErrPullRequestAlreadyScheduledToAutoMerge) Error() string {
	return fmt.Sprintf("pull request is already scheduled to auto merge when checks succeed [pull_id: %d]", err.PullID)
}
```

The comment above `Error` was deleted because it is also not present above the other `Error` methods.